src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -393,7 +393,7 @@ export class DataViewsService {
     const savedObjects = await this.savedObjectsClient.find({
       fields: ['title'],
       search,
-      searchFields: ['title'],
+      searchFields: ['title', 'name'],
```

Comment:
This code could be better - add the `name` field to the `fields` array - We should be displaying the `name` to identify a data view, failing that, the title.

Reply:
The line immediately afterwards loads the DataView from saved objects and returns the entire structure (`name` included). IMHO, we don't really even need to fetch the `title` field, we just need the `id`.